docs/resources/Channel.md

```diff
@@ -513,17 +513,17 @@ Update a channel's settings. Requires the `MANAGE_CHANNELS` permission for the g
-| nsfw                  | boolean                                                                | whether the channel is nsfw                                                                                                                                                     | Text         |
-| rate_limit_per_user   | integer                                                                | amount of seconds a user has to wait before sending another message (0-21600); bots, as well as users with the permission `manage_messages` or `manage_channel`, are unaffected | Text         |
-| bitrate               | integer                                                                | the bitrate (in bits) of the voice channel; 8000 to 96000 (128000 for VIP servers)                                                                                              | Voice        |
-| user_limit            | integer                                                                | the user limit of the voice channel; 0 refers to no limit, 1 to 99 refers to a user limit                                                                                       | Voice        |
-| permission_overwrites | array of [overwrite](#DOCS_RESOURCES_CHANNEL/overwrite-object) objects | channel or category-specific permissions                                                                                                                                        | All          |
-| parent_id             | snowflake                                                              | id of the new parent category for a channel                                                                                                                                     | Text, Voice  |
+| Field                 | Type                                                                    | Description                                                                                                                                                                     | Channel Type |
+| --------------------- | ----------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------ |
+| name                  | string                                                                  | 2-100 character channel name                                                                                                                                                    | All          |
+| position              | ?integer                                                                | the position of the channel in the left-hand listing                                                                                                                            | All          |
```

Comment:
all these being optional is untrue for PUT, so these *must* be marked as required. Line 512 defines these as all optional for PATCH already..

Reply:
Right now `name` isn't nullable for this patch endpoint and will return a `400` error with the response seen below, blanket marking all of these fields as nullable doesn't seem to be a fit here unless that's unintended behaviour and even then if the intended behaviour of setting `name` as `null` for this patch endpoint is for it to ignore the change as opposed to setting the name to some sort of default (e.g. "general") then it doesn't seem like it'd make sense to call this field "nullable".
```json
{
  "name": [
    "This field is required"
  ]
}
```